platform/kobo/fmon/README.txt

```diff
@@ -1,19 +1,23 @@
 Installation instructions for Kobo:
 
 
-Newer installation based on Start Menu:
-- Install "Start Menu" (http://www.mobileread.com/forums/showthread.php?t=255978)
-- Extract the koreader folder of the zip into the ".adds" directory (both KoboRoot.tgz and the koreader.png file are only needed for the older installation based on Filemonitor. See below.).
+Main installation method based on Start Menu:
+- Install "Start Menu" (https://www.mobileread.com/forums/showthread.php?t=293804)
+- Extract the koreader folder of the zip into the ".adds" directory (the koreader.png file are only needed for the alternate installation methods based on fmon. See below.).
 
 Select koreader from the Start Menu to start.
 
+----
 
-Older installation based on Filemonitor:
-- Put the image included in the zip (called "koreader.png") in the main folder of your kobo and disconnect it from your computer. Open the image on the reader, go back to the home and then, just to be extra-safe, reboot it.
-- Install "Files Monitor" (http://www.mobileread.com/forums/showthread.php?t=218283).
-- Extract the remaining content of the zip into the ".kobo" directory (both KoboRoot.tgz and the koreader folder).
+Alternate installation method based on fmon:
+- Put the image included in the zip (called "koreader.png") in an "icons" folder created in the main folder of your Kobo and disconnect it from your computer. Add the icon to a collection, set the view to List View and open the icon, then restart your device.
+- Install "fmon" (https://github.com/baskerville/fmon).
+- Extract the remaining content of the zip into the ".adds" directory (the koreader folder).
 
-Selecting the KOReader icon in your home, you will be able to launch KOReader. Simple enough, isn't it? Just be sure to process the image properly (follow those steps carefully) if it's the first time you install koreader, if you made a factory reset or if you deleted it in some way, otherwise you could end up launching koreader automatically at every boot, being unable to exit it without a factory reset.
+Selecting the KOReader icon in your home, you will be able to launch KOReader. Simple enough, isn't it? Just be sure to process the image properly (follow those steps carefully) if it's the first time you install koreader, after a factory reset or if you deleted it in some way, in order to make sure KOReader starts properly.
```

Comment:
Seems a bit wordy in the first sentence. :-P

Quick suggestion:

Select the KOReader icon in your home to launch KOReader. Simple enough, isn't it? Just be sure to process the image properly. Follow these steps carefully if it's your first time installing KOReader, after a factory reset, or if you deleted it in some way. This will ensure KOReader starts properly.

Reply:
Yeah, that's my curse ^^. Thanks for the suggestion, will use it.